velox/exec/tests/TableWriteTest.cpp

```diff
@@ -149,16 +160,58 @@ class TableWriteTest : public HiveConnectorTestBase {
 
   void SetUp() override {
     HiveConnectorTestBase::SetUp();
-    parquet::registerParquetReaderFactory();
```

Comment:
This change will cause the Parquet format to be skipped in the tests since we are not registering Parquet anymore.